js/header.js

```diff
@@ -0,0 +1,9 @@
+navbar = document.getElementsByTagName('nav')[0];
+
+window.addEventListener('scroll', function(e) {
```

Comment:
For when the page scrolls? I didn't know you could do that.

Reply:
For the width I meant

Reply:
I guess I could, but then the floating class will still get added to the element even when the class has no effect. That would just seem a little counter-intuitive because the navbar might not be floating even if it has the floating class.